### PR TITLE
Installation section simplification

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

The installation section contained a lot of old paragraphs and stuff that felt like digressions instead of instructions. I thinned down the docs and kept the important stuff. Also put "What does kickstart do" before the installation, so that users can be aware of this (it was a scroll away before, hidden)

### CODE REVIEW

File: packaging/installer/README.md [hunk 1]
```diff
@@ -6,150 +6,8 @@ Netdata is very flexible and can be used to monitor all kinds of infrastructure.
...
-
-- Protect yourself from the rare instance when major bugs slip through our testing and negatively affect a Netdata installation
-- Retain more control over the Netdata version you use
-
-### Anonymous statistics
-
-Starting with v1.30, Netdata collects anonymous usage information by default and sends it to a self-hosted PostHog instance within the Netdata infrastructure. Read about the information collected, and learn how to-opt, on our [anonymous statistics](/docs/netdata-agent/configuration/anonymous-telemetry-events.md) page.
-
-The usage statistics are _vital_ for us, as we use them to discover bugs and prioritize new features. We thank you for
-_actively_ contributing to Netdata's future.
```
Comment (Ferroin) [inline packaging/installer/README.md:79]: This _NEEDS_ to be kept, period. Users should not have to go digging to find out that we do anonymous statistics collection.

File: packaging/installer/README.md [hunk 2]
```diff
@@ -6,150 +6,8 @@ Netdata is very flexible and can be used to monitor all kinds of infrastructure.
...
-   affect many projects other than just Netdata, and there are unfortunately a number of other services out there
-   that do not provide IPv6 connectivity, so taking this route is likely to save you time in the future as well.
-2. If you are using a system that we publish native packages for (see our [platform support
-   policy](/docs/netdata-agent/versions-and-platforms.md) for more details),
-   you can manually set up our native package repositories as outlined in our [native package install
-   documentation](/packaging/installer/methods/packages.md). Our official
-   package repositories do provide service over IPv6, so they work without issue on hosts without IPv4 connectivity.
-3. If neither of the above options work for you, you can still install using our [offline installation
-   instructions](/packaging/installer/methods/offline.md), though
-   do note that the offline install source must be prepared from a system with IPv4 connectivity.
```
Comment (Ferroin) [inline packaging/installer/README.md:104]: This section is still accurate and is absolutely relevant to support. It still comes up on occasion with users who have not read the documentation, it’s non-obvious, and if a user runs into this they _CANNOT INSTALL NETDATA WITHOUT RESOLVING IT_.

File: packaging/installer/methods/kickstart.md [hunk 3]
```diff
@@ -5,17 +5,29 @@ import TabItem from '@theme/TabItem';
...
+## What does `kickstart.sh` do?
+
+The `kickstart.sh` script does the following after being downloaded and run using `sh`:
+
+- Determines what platform you are running on.
+- Checks for an existing installation, and if found updates that instead of creating a new install.
+- Attempts to install Netdata using our [official native binary packages](/packaging/installer/methods/packages.md).
+- If there are no official native binary packages for your system (or installing that way failed), tries to install using a [static build of Netdata](/packaging/makeself/README.md) if one is available.
+- If no static build is available, installs required dependencies and then attempts to install by building Netdata locally (by downloading the sources and building them directly).
+- Installs `netdata-updater.sh` to `cron.daily`, so your Netdata installation will be updated with new nightly versions, unless you override that with an [optional parameter](#optional-parameters-to-alter-your-installation).
```
Comment (Ferroin) [inline packaging/installer/methods/kickstart.md:23]: This arguably needs to be updated, as it’s description is not entirely accurate (it only describes the default case, which is not always the case).

**Comment by Ancairon** [comment]
I'd argue all the note sections need to be on the main README, they can be a separate file IMO

File: packaging/installer/methods/kickstart.md (hunk 3 above)
Comment (Ancairon) [reply packaging/installer/methods/kickstart.md:23]: can you update it? I am not sure I know the rest of the cases / can't know them

File: packaging/installer/README.md (hunk 1 above)
Comment (Ancairon) [reply packaging/installer/README.md]: I added this back in the file

File: packaging/installer/README.md (hunk 2 above)
Comment (Ancairon) [reply packaging/installer/README.md:104]: made a new file for this, that will be inside the linux section in our docs

File: packaging/installer/UPDATE.md [hunk 4]
```diff
@@ -106,8 +106,3 @@ The following configuration options are currently supported:
...
 - `NETDATA_MAJOR_VERSION_UPDATES`: If set to a value other than 0, then new major versions will be installed without user confirmation. Must be set to a non-zero value for automated updates to install new major versions.
 - `NETDATA_NO_SYSTEMD_JOURNAL`: If set to a value other than 0, skip attempting to install the  `netdata-plugin-systemd-journal` package on supported systems on update. The updater will install this optional package by default on supported systems if this option is not set. It only affects systems using native packages.
-
-### Updates on hosts without IPv4 connectivity
-
-The update process outlined above suffers from the same issues that installing on hosts without IPv4 connectivity does, and requires similar workarounds.
-For more details, check [the explanation in our install documentation](/packaging/installer/README.md#installs-on-hosts-without-ipv4-connectivity).
```
[the remark this answers is not in the document]
Comment (Ancairon) [reply packaging/installer/UPDATE.md:113]: I think that this does not serve a real purpose, it points to the installing section, when the goal is to update the Agent. If someone has installed the Agent on a non-IPv4 host, then he will probably know what to do to update said install

File: packaging/installer/methods/kickstart.md (hunk 3 above)
Comment (Ferroin) [reply packaging/installer/methods/kickstart.md:23]: Yes, I’ll plan to do a new PR in the next few days to update it appropriately.

**Pull request merged**